src/app/src/MeetTheFish.js

```diff
@@ -1,20 +1,21 @@
 import React from 'react';
 import styled from 'styled-components';
-import { Flex, Box, Heading, Text } from 'rebass';
+import { Flex, Box } from 'rebass';
+import { Heading, Text } from '../src';
```

Comment:
I think we would benefit by having a single file from where the custom styled components can be imported (similar to `main.css`'s role) which is more clear than generally importing from a folder (`../src`). That, or import them individually like `import Heading from './Heading'`

Reply:
That makes a ton of sense! I'll take a stab at that instead. Blaster does something similar to what you're describing.

Reply:
What do you think would be a good name for that file?

Reply:
Great! Hm `custom-styled-components.js`?